data/json/itemgroups/food_service.json

```
@@ -976,5 +976,49 @@
       { "item": "stout", "prob": 10, "container-item": "keg" },
       { "item": "imperial_stout", "prob": 5, "container-item": "keg" }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "butcher_raw_meat",
+    "items": [
+      [ "meat", 8 ],
+      [ "lunchmeat", 6 ],
+      [ "offal", 10 ],
+      [ "hotdogs_frozen", 6 ],
+      { "item": "lung", "prob": 1, "charges": 2 },
+      { "item": "kidney", "prob": 1, "charges": 2 },
+      { "item": "liver", "prob": 1, "charges": 1 },
```

```suggestion
      { "item": "liver", "prob": 4, "charges": 5 },
```
Liver gets higher probability than the other offal items because it has a much noticeably higher probability in the harvest lists.